diff --git a/wysiwyg.install b/wysiwyg.install
index 1874e5b..4efae8e 100644
--- a/wysiwyg.install
+++ b/wysiwyg.install
@@ -91,7 +91,7 @@ function wysiwyg_enable() {
   // Disable conflicting, obsolete editor integration modules whenever this
   // module is enabled. This is crude, but the only way to ensure no conflicts.
   module_disable(array(
-    'ckeditor',
+    // 'ckeditor', // Commented ckeditor out by Kanopi, as it is used in Mukurtu for Scald in text fields.
     'editarea',
     'editonpro',
     'editor',
